ext/ZygoteExt.jl

```diff
@@ -0,0 +1,97 @@
+module ZygoteExt
+
+using Zygote: pullback
+using ZygoteRules: @adjoint
+using SciMLBase: ODESolution, issymbollike, sym_to_index, remake, getobserved
+
+@adjoint function getindex(VA::ODESolution, i::Int, j::Int)
+    function ODESolution_getindex_pullback(Δ)
+        du = [m == j ? [i == k ? Δ : zero(VA.u[1][1]) for k in 1:length(VA.u[1])] :
+              zero(VA.u[1]) for m in 1:length(VA.u)]
+        dp = zero(VA.prob.p)
+        dprob = remake(VA.prob, p = dp)
+        du, dprob
+        T = eltype(eltype(VA.u))
+        N = length(VA.prob.p)
+        Δ′ = ODESolution{
+            T,
+            N,
+            typeof(du),
+            Nothing,
+            Nothing,
+            typeof(VA.t),
+            typeof(VA.k),
+            typeof(dprob),
+            typeof(VA.alg),
+            typeof(VA.interp),
+            typeof(VA.destats),
+            typeof(VA.alg_choice),
+        }(du,
+            nothing,
+            nothing,
+            VA.t,
+            VA.k,
+            dprob,
+            VA.alg,
+            VA.interp,
+            VA.dense,
+            0,
+            VA.destats,
+            VA.alg_choice,
+            VA.retcode)
+        (Δ′, nothing, nothing)
```

Comment:
This is the wrong formatting.

Reply:
It won't revert this if you run the formatter, so put it all to one line and then reformat with the right format.

Reply:
Changed, maybe fixed?